components/AppLayout/AppHeader.js

```diff
@@ -1,66 +1,244 @@
+    letterSpacing: 0.75,
+    color: theme.palette.secondary[300],
+    [theme.breakpoints.up('md')]: {
+      color: theme.palette.secondary[500],
+      padding: '0 10px',
+    },
+  },
+  activeTab: {
+    color: theme.palette.secondary[500],
+    [theme.breakpoints.up('md')]: {
+      color: theme.palette.primary.main,
+    },
+  },
+  menuToggleButton: {
+    width: MENU_BUTTON_WIDTH,
+    height: TABS_HEIGHT,
+    display: 'flex',
+    justifyContent: 'center',
+    alignItems: 'center',
+    background: theme.palette.secondary[300],
+    color: theme.palette.common.white,
+  },
+  profileMenu: {
+    marginTop: 50,
+    backgroundColor: theme.palette.secondary.main,
+    color: theme.palette.common.white,
+    overflow: 'inherit',
+  },
+  divider: {
+    backgroundColor: theme.palette.secondary[400],
+  },
+  listIcon: {
+    color: theme.palette.secondary[300],
+    minWidth: 0,
+    paddingRight: 8,
+  },
+  login: {
+    borderRadius: 70,
+    border: `1px solid ${theme.palette.secondary[500]}`,
+  },
+}));
+
+const Links = ({ classes }) => (
+  <div className={classes.tabs}>
+    <NavLink
+      href="/articles"
+      className={classes.tab}
+      activeClassName={classes.activeTab}
+    >
+      {t`Collected Messages`}
+    </NavLink>
+    <NavLink
+      href="/replies"
+      className={classes.tab}
+      activeClassName={classes.activeTab}
+    >
+      {t`Replies`}
+    </NavLink>
+    <NavLink
+      external
+      href={EDITOR_FACEBOOK_GROUP}
+      className={cx(classes.tab, 'hidden-xs')}
+    >{t`Editor forum`}</NavLink>
+    <NavLink
+      external
+      href={PROJECT_HACKFOLDR}
+      className={classes.tab}
+    >{t`About`}</NavLink>
+  </div>
+);
 
-        @media screen and (min-width: 768px) {
-          .root {
-            padding: 0 40px;
-          }
-        }
-      `}</style>
+function AppHeader({ onMenuButtonClick, user, openLoginModal, logout }) {
+  const [anchor, setAnchor] = useState(false);
+  const classes = useStyles();
+  const theme = useTheme();
+
+  const openProfileMenu = useCallback(e => setAnchor(e.currentTarget), [
+    anchor,
+  ]);
+  const closeProfileMenu = useCallback(() => setAnchor(null), [anchor]);
+
+  return (
+    <header className={classes.root}>
+      <div className={classes.top}>
+        <div className={classes.flex}>
+          <a href="/">
+            <img
+              className={classes.logo}
+              src={mobileLogo}
+              srcSet={`${desktopLogo} ${theme.breakpoints.values.md}w`}
+              alt=""
+            />
+          </a>
+          <Box display={['none', 'none', 'inline']}>
+            <Links classes={classes} />
+          </Box>
+        </div>
+        {/* GlobalSearch not fully implemented yet */}
+        {/* <GlobalSearch />*/}
+        <Box display={['none', 'none', 'block']}>
+          {user?.name ? (
+            <>
+              <Widgets.Avatar user={user} size={40} onClick={openProfileMenu} />
+              <Menu
```

Comment:
We may need to hunt down the root cause of this warning:
![image](https://user-images.githubusercontent.com/108608/78017769-5efe8200-737f-11ea-841c-a5a9cbb128bd.png)